src/rsz/src/Resizer.cc

```diff
@@ -960,6 +954,13 @@ Resizer::replaceCell(Instance *inst,
     sta_->replaceCell(inst, replacement_cell1);
     designAreaIncr(area(replacement_master));
 
+    // Legalize the position of the instance in case it leaves the die
+    if (parasitics_src_ == ParasiticsSrc::global_routing) {
+      opendp_->legalCellPos(db_network_->staToDb(inst));
+    } else if (parasitics_src_ == ParasiticsSrc::global_routing) {
+      logger_->error(RSZ, 91, "Opendp was not initialized before resized an instance");
+    }
```

Comment:
this doesn't look like it does anything since `parasitics_src_ == ParasiticsSrc::global_routing` is already checked

Reply:
I fixed the mistake, thanks